mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -547,8 +547,9 @@ server_connect_dialog.dropbox.load_account = Load account
 server_connect_dialog.dropbox.username = $[server_connect_dialog.username]
 server_connect_dialog.dropbox.account = $[server_connect_dialog.account_alias]
 adb.android_devices = Android
-adb.no_devices = No device discovered
 adb.android_disabled = Android disabled
+adb.no_devices = No device discovered
+adb.searching = Searching...
```

Comment:
we already have "Loading..." translated, how about using it instead?

Reply:
I thought that 'searching' better describes what it does - I can use 'loading', but 'searching' is better for me. If you insist I can change that to 'loading'.

Reply:
I don't see much difference and considering that we don't get translation updates so often, it's better to avoid adding new ones unless we really need to

Reply:
PR updated